pkg/etcd/util.go

```diff
@@ -67,14 +67,11 @@ func GetEtcdClient(ctx context.Context, certificates *Certificates, endpoints ..
 // If no endpoints are provided, getEndpoints is called to provide defaults.
 func getClientConfig(ctx context.Context, certificates *Certificates, endpoints ...string) (*clientv3.Config, error) {
 	config := &clientv3.Config{
-		Endpoints:            endpoints,
-		Context:              ctx,
-		DialTimeout:          2 * time.Second,
-		DialKeepAliveTime:    30 * time.Second,
-		DialKeepAliveTimeout: 10 * time.Second,
-		AutoSyncInterval:     10 * time.Second,
-		Logger:               zap.L().Named("etcd-client"),
-		PermitWithoutStream:  true,
+		Endpoints:   endpoints,
+		Context:     ctx,
+		DialTimeout: 5 * time.Second,
```

Comment:
why is it changed?

Reply:
This causes issues if you just want to query etcd for values as its trying to reach all peers